packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -99,6 +99,8 @@ function loadStory(storyId, post, restore, clearHistory) {
     storyId: storyId,
     title,
     status: statusFormat,
+    // @todo Temporary, check for an alternative.
+    isNew: status === 'auto-draft',
```

Comment:
In Gutenberg, the check for `isNew` happens by checking the status for `auto-draft`, in our editor, however, `auto-draft` is overwritten to `draft`. I suppose it makes it easier to just handle the `draft` status in the code, however, now it's not possible to check if the post is new by that. Added `isNew` here for now, wondering if there's a better alternative 🤔

Reply:
> in our editor, however, `auto-draft` is overwritten to `draft`

Can you remind me where this happens?

I don't think that's right and might be why the "All Stories" page is easily flooded with draft stories instead of auto-draft ones (which will cleaned up by WordPress automatically):

![Screenshot 2022-08-22 at 16 43 24](https://user-images.githubusercontent.com/841956/185949563-f413267b-b7b6-4572-a777-aa30e5b2e563.png)

Reply:
Hm, could be. Not sure when it was added exactly:
https://github.com/GoogleForCreators/web-stories-wp/blob/96f067b0283184f672fbf0268b13285f945a2918/packages/story-editor/src/app/story/effects/useLoadStory.js#L76

Reply:
Sure!